model/plugin_event_data.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package model
+
+import (
+	"encoding/json"
+	"io"
+)
+
+// PluginEventData used to send plugin data to other clusters.
+type PluginEventData struct {
+	PluginId            string `json:"plugin_id"`
```

Comment:
nit: `Plugin*` stutters -- is it reasonable to just use `Id` and `FileStorePath`?